lib/net.c

```diff
@@ -43,16 +45,43 @@
 static struct net_config config;
 
 /*
- * XXX The secret key of the RSS hash must be random
- * in order to avoid hackers to know it.
+ * The secret key of the RSS hash (RSK) must be random in order
+ * to prevent hackers from knowing it.
+ * To validate if the key generated is reasonable, the
```

Comment:
Please put the comment in this line and below right before the definition of the variable: ``min_num_set_bits``.

Reply:
Please keep the comment ``The secret key of the RSS hash (RSK) must be random in order to prevent hackers from knowing it.`` above the definition of ``uint8_t default_rss_key[GATEKEEPER_RSS_KEY_LEN];``. The remaining lines of comments are describing the two variables, i.e., ``uint16_t min_num_set_bits`` and ``uint16_t max_num_set_bits``.